### PR TITLE
S05US41: lost password

### PR DESCRIPTION
**Pull request by JuMn88**
Adding a Reset Password Feature

### CODE REVIEW

**Review by Wowlfy — APPROVED** [review]

File: src/Controller/ResetPasswordController.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+    public function __construct(ResetPasswordHelperInterface $resetPasswordHelper)
+    {
+        $this->resetPasswordHelper = $resetPasswordHelper;
+    }
+
+    /**
+     * Display & process form to request a password reset.
+     *
+     * @Route("", name="app_forgot_password_request")
```
Comment (LucyMerlier) [inline src/Controller/ResetPasswordController.php:38]: dont just put an empty string here, write "/" instead

File: templates/reset_password/check_email.html.twig [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
```
Comment (LucyMerlier) [inline templates/reset_password/check_email.html.twig:1]: this extends base.html.twig directly?

File: templates/reset_password/request.html.twig [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html.twig' %}
```
Comment (LucyMerlier) [inline templates/reset_password/request.html.twig:1]: this extends base.html.twig directly?

File: templates/reset_password/reset.html.twig [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html.twig' %}
```
Comment (LucyMerlier) [inline templates/reset_password/reset.html.twig:1]: this extends base.html.twig directly?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Erdnaxela45 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**